packages/cli/test/unit/services/hooks.service.test.ts

```diff
@@ -0,0 +1,164 @@
+import { UserService } from '@/services/user.service';
+import { HooksService } from '@/services/hooks.service';
+import { User } from '@/databases/entities/User';
+import { mockInstance } from '../../shared/mocking';
+import { v4 as uuid } from 'uuid';
+import { AuthService } from '@/auth/auth.service';
+import { CredentialsRepository } from '@/databases/repositories/credentials.repository';
+import { SettingsRepository } from '@/databases/repositories/settings.repository';
+import { UserRepository } from '@/databases/repositories/user.repository';
+import { WorkflowRepository } from '@/databases/repositories/workflow.repository';
+import type { Response } from 'express';
+
+let hooksService: HooksService;
+
+const mockedUser = Object.assign(new User(), {
+	id: uuid(),
+	password: 'passwordHash',
+	mfaEnabled: false,
+	mfaSecret: 'test',
```

Comment:
this is why I keep complaining about `Object.assign`: There is no `mfaSecret` on `User` anymore. 
I'll take care of this.